### PR TITLE
Actually use the connection's set dataEncoding

### PR DESCRIPTION
**Pull request by stewarttylerr**
@crash83k This should hopefully fix https://github.com/stewarttylerr/ftpserver/issues/8
Just a simple issue of not actually setting the correct dataEncoding.

### CODE REVIEW

File: lib/ftp/commands/stor.js [hunk 1]
```diff
@@ -24,8 +24,9 @@ export default function (thisCmd, fileName) {
         this.reply(552);
         reject(err);
       });
-
-      this.dataSocket.pipe(stream);
+      this.dataSocket.on('data', (data) => {
+        stream.write(data, this.dataEncoding);
+      });
```
Comment (crash83k) [inline lib/ftp/commands/stor.js]: I'm doing this as well, and it's working properly.
My code is slightly different, but should work exactly as yours does:
```
_this.dataSocket.on('data', function (data) {
        stream.write(Buffer.from(data, 'binary'));
      });
```

File: lib/ftp/connection.js [hunk 2]
```diff
@@ -26,6 +26,7 @@ export default class Connection {
     this.fs = new FileSystem();
     this.lastCommand = moment.utc();
 
+    this.dataEncoding = 'utf8';
```
Comment (crash83k) [inline lib/ftp/connection.js]: Is this a default, in the case the client doesn't set the encoding?

**Comment by crash83k** [comment]
Looks good. But you're setting `this.dataEncoding` to `null`. That's just a default and won't affect the `this.dataSocket` encoding, correct?

File: lib/ftp/connection.js (hunk 2 above)
Comment (stewarttylerr) [inline lib/ftp/connection.js]: Yeah, just default to utf8

**Pull request merged**